### PR TITLE
app-vim/*: EAPI8 bumps

### PR DESCRIPTION
**Pull request by mm1ke**
Hi,

This are some simple EAPI8 bumps for app-vim plugins. Also updates `bufexplorer` to the latest version and removes the live ebuild (i guess it's not really relevant here).
Please review.

### CODE REVIEW

File: app-vim/csscomplete/csscomplete-1.0-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+DESCRIPTION="vim plugin: CSS 3 omni complete function"
+HOMEPAGE="https://github.com/othree/csscomplete.vim"
+SRC_URI="https://github.com/othree/${PN}.vim/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="vim.org"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
+
+S="${WORKDIR}/${PN}.vim-${PV}"
```
Comment (SoapGentoo) [inline app-vim/csscomplete/csscomplete-1.0-r1.ebuild:16]: put this right after `SRC_URI`

File: app-vim/csound-syntax/csound-syntax-20160804.ebuild [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
+# Copyright 1999-2023 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=6
 
 inherit vim-plugin
 
 DESCRIPTION="vim plugin: set of tools for editing Csound files with vim"
-HOMEPAGE="https://github.com/luisjure/csound"
+HOMEPAGE="https://github.com/luisjure/csound-vim"
```
Comment (SoapGentoo) [inline app-vim/csound-syntax/csound-syntax-20160804.ebuild:9]:
```suggestion
HOMEPAGE="https://github.com/luisjure/csound-vim"

```

File: app-vim/csound-syntax/csound-syntax-20160804-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit vim-plugin
+
+DESCRIPTION="vim plugin: set of tools for editing Csound files with vim"
+HOMEPAGE="https://github.com/luisjure/csound-vim"
```
Comment (SoapGentoo) [inline app-vim/csound-syntax/csound-syntax-20160804-r1.ebuild:9]:
```suggestion
HOMEPAGE="https://github.com/luisjure/csound-vim"

```

File: app-vim/cmdalias/cmdalias-3.0-r2.ebuild [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit vim-plugin
+
+DESCRIPTION="vim plugin: library for alias creation"
+HOMEPAGE="https://www.vim.org/scripts/script.php?script_id=746"
```
Comment (SoapGentoo) [inline app-vim/cmdalias/cmdalias-3.0-r2.ebuild:9]:
```suggestion
HOMEPAGE="https://www.vim.org/scripts/script.php?script_id=746"

```

File: app-vim/cctree/cctree-1.61-r2.ebuild [hunk 5]
```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit vim-plugin
+
+DESCRIPTION="vim plugin: Cscope based source-code browser and code flow analysis tool"
+HOMEPAGE="https://sites.google.com/site/vimcctree/"
```
Comment (SoapGentoo) [inline app-vim/cctree/cctree-1.61-r2.ebuild:9]:
```suggestion
HOMEPAGE="https://sites.google.com/site/vimcctree/"

```

File: app-vim/calendar/calendar-2.9-r2.ebuild [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit vim-plugin
+
+DESCRIPTION="vim plugin: calendar window"
+HOMEPAGE="https://www.vim.org/scripts/script.php?script_id=52"
```
Comment (SoapGentoo) [inline app-vim/calendar/calendar-2.9-r2.ebuild:9]:
```suggestion
HOMEPAGE="https://www.vim.org/scripts/script.php?script_id=52"

```

File: app-vim/bufexplorer/bufexplorer-7.4.25.ebuild [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+EAPI=8
+
+inherit vim-plugin
+
+DESCRIPTION="vim plugin: easily browse vim buffers"
+HOMEPAGE="
+  https://www.vim.org/scripts/script.php?script_id=42
+  https://github.com/jlanzarotta/bufexplorer"
+SRC_URI="https://github.com/jlanzarotta/${PN}/archive/v.${PV}.tar.gz -> ${P}.tar.gz"
```
Comment (SoapGentoo) [inline app-vim/bufexplorer/bufexplorer-7.4.25.ebuild:12]:
```suggestion
SRC_URI="https://github.com/jlanzarotta/${PN}/archive/v.${PV}.tar.gz -> ${P}.tar.gz"

```

File: app-vim/bnf-syntax/bnf-syntax-1.2-r3.ebuild [hunk 8]
```diff
@@ -0,0 +1,20 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit vim-plugin
+
+DESCRIPTION="vim plugin: BNF file syntax highlighting"
+HOMEPAGE="https://www.vim.org/scripts/script.php?script_id=250"
```
Comment (SoapGentoo) [inline app-vim/bnf-syntax/bnf-syntax-1.2-r3.ebuild:9]:
```suggestion
HOMEPAGE="https://www.vim.org/scripts/script.php?script_id=250"

```

File: app-vim/alternate/alternate-2.18-r3.ebuild [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+EAPI=8
+
+inherit vim-plugin
+
+DESCRIPTION="vim plugin: quickly switch between .c and .h files"
+HOMEPAGE="
+	https://github.com/vim-scripts/a.vim
+	https://www.vim.org/scripts/script.php?script_id=31"
+
```
Comment (SoapGentoo) [inline app-vim/alternate/alternate-2.18-r3.ebuild:12]:
```suggestion
```

File: app-vim/alternate/alternate-2.18-r3.ebuild [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+file and the associated .h file. There is also :AS to split windows and
+:AV to split windows vertiically."
+
+VIM_PLUGIN_HELPFILES="${PN}.txt"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-hh-cc-alternation.patch
+)
+
+S="${WORKDIR}/a.vim-${PV}"
```
Comment (SoapGentoo) [inline app-vim/alternate/alternate-2.18-r3.ebuild:31]: right after `SRC_URI`

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

**Comment by mm1ke** [comment]
@SoapGentoo thanks for the review :) I've updated all your suggestions.

**Comment by mm1ke** [comment]
I've updated most vim plugin for `EAPI8` now. Some ebuilds, which have only unstable keywords, were updated directly to `EAPI8`. I've also made a few version bumps.

**Pull request closed without merging**